### PR TITLE
Add RancherOS section to README

### PR DESCRIPTION
**Pull request by pdostal**
As we now have [pasta.sh.cvut.cz](https://pasta.sh.cvut.cz/) I'm sharing the steps describing the deployment on [RancherOS](https://rancher.com/docs/os/v1.x/en/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,22 @@ Assuming your data is in `/srv/pasta/` you can do
...
+
+Let's assume we have `/dev/sda` for the system and `/dev/sdb` for data.
+
+* Prepare persistent storage for data
+* Install the system with given [`cloud-init.yaml`](cloud-init.yaml.example) to system storage
+* Configure your proxy and enojoy!
+
+```bash
+$ sudo parted /dev/sdb
+$ # mktable - gpt - mkpart - 0% - 100%
```
Comment (tuxjz) [inline README.md:33]: missing partition number after mkpart command

File: cloud-init.yaml.example [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+rancher:
+  network:
+    dns:
+      nameservers:
+        - 8.8.8.8
+        - 1.1.1.1
+    interfaces:
+      eth0:
+        addresses:
+          - 172.16.0.2/24
```
Comment (tuxjz) [inline cloud-init.yaml.example:29]:
``` suggestion
          - 192.0.2.2/24
```

File: cloud-init.yaml.example [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          - 2001:123:456::2/64
+        gateway: 172.16.0.1
```
Comment (tuxjz) [inline cloud-init.yaml.example:31]:
``` suggestion
        gateway: 192.0.2.1
```

File: cloud-init.yaml.example [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  network:
+    dns:
+      nameservers:
+        - 8.8.8.8
+        - 1.1.1.1
+    interfaces:
+      eth0:
+        addresses:
+          - 172.16.0.2/24
+          - 2001:123:456::2/64
```
Comment (tuxjz) [inline cloud-init.yaml.example:30]:
``` suggestion
          - 2001:db8::2/64
```

File: cloud-init.yaml.example [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        gateway: 172.16.0.1
+        gateway_ipv6: 2001:123:456::1
```
Comment (tuxjz) [inline cloud-init.yaml.example:32]:
``` suggestion
        gateway_ipv6: 2001:db8::1
```

**Review by tuxjz — CHANGES_REQUESTED** [review]

**Review by tuxjz — APPROVED** [review]
Good job 🙃 🥂

**Comment by grisu48** [comment]
LGTM :rocket: Thanks for the PR! :slightly_smiling_face:

**Pull request merged**